Lib/test/test_threading.py

```diff
@@ -86,6 +87,33 @@ def tearDown(self):
 
 class ThreadTests(BaseTestCase):
 
+    @cpython_only
```

Comment:
Hmm, should this be marked as `cpython_only`? I don't think this involves any implementation detail.

Reply:
The test monkey-patches the _counter attribute which is a private attribute.

Reply:
Also, I'm fine with other Python implementation using a different naming when name argument is omitted.

Reply:
Good point, sounds good.